### ISSUE TITLE
[Question]: How does Inpaint ControlNet with Mask Upload work?

### ISSUE THREAD

**Issue by Haoming02** [Mikubill/sd-webui-controlnet#2625]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

In `txt2img`, I tried to use the **Inpaint** ControlNet with **Mask Upload**. But it only generates mess:

- **inpaint_only**
![image (3)](https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/560b6df5-5297-4dd5-959d-379f0fa3aaec)

- **inpaint_global_harmonious**
![image (4)](https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/8f66ee94-0f64-4d35-a90e-d763c8afec5c)

- **inpaint_only+lama**
![image (5)](https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/d857562e-6484-4eda-b270-6d5de37a1a06)

### Steps to reproduce the problem

1. Go to `txt2img`
2. Open ControlNet
3. Upload a source image
4. Enable **Mask Upload**
5. Upload a black & white mask image
6. Set `Control Type` to **Inpaint**
7. Fill out the `txt2img` parameters
8. Generate

### What should have happened?

The following is the generated output using Automatic1111's built-in **Inpaint upload** mode in `img2img` tab:
<p align="center"><img src="https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/4d9b272f-352e-4f24-bc46-2eacfe80f257"></p>

### Commit where the problem happens

- **Webui:** v1.7.0
- **ControlNet:** [416c345](https://github.com/Mikubill/sd-webui-controlnet/commit/416c345072c9c2066101e225964e3986abe6945e)

### Command Line Arguments

```Shell
--xformers --no-half-vae
```

### List of enabled extensions

N/A

### Console logs

```Shell
2024-02-06 15:15:46,015 - ControlNet - INFO - unit_separate = False, style_align = False 29/29 [00:38<00:00,  1.11s/it]
2024-02-06 15:15:46,170 - ControlNet - INFO - Loading model: control_v11p_sd15_inpaint [ebff9138]
2024-02-06 15:15:46,608 - ControlNet - INFO - Loaded state_dict from [C:\Users\Michael.C\Documents\GitHub\stable-diffusion-webui\models\ControlNet\control_v11p_sd15_inpaint.pth]
2024-02-06 15:15:46,609 - ControlNet - INFO - controlnet_default_config
2024-02-06 15:15:47,891 - ControlNet - INFO - ControlNet model control_v11p_sd15_inpaint [ebff9138](ControlModelType.ControlNet) loaded.
2024-02-06 15:15:47,891 - ControlNet - INFO - using inpaint as input
2024-02-06 15:15:47,899 - ControlNet - INFO - Using preprocessor: inpaint_only
2024-02-06 15:15:47,899 - ControlNet - INFO - preprocessor resolution = -1
2024-02-06 15:15:47,976 - ControlNet - INFO - ControlNet Hooked - Time = 1.9622855186462402
  0%|                                                                                           | 0/32 [00:00<?, ?it/s]2024-02-06 15:15:48,765 - ControlNet - INFO - ControlNet used torch.float32 VAE to encode torch.Size([1, 4, 128, 192]).
100%|██████████████████████████████████████████████████████████████████████████████████| 32/32 [00:57<00:00,  1.79s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 32/32 [00:57<00:00,  1.79s/it]
```

### Additional information

- **Source Image**
![image (7)](https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/ebab01df-1452-4c83-919e-42575f679dd8)

- **Mask**
![image (8)](https://github.com/Mikubill/sd-webui-controlnet/assets/73768377/ec8b7943-85fe-43c8-93c2-5e1b22e05e15)

**Comment by Haoming02** [comment]
*(**Browser** is `Edge`)*
